zk.js/src/transaction/transactionParameters.ts

```diff
@@ -1144,11 +1157,14 @@ export class TransactionParameters implements transactionParameters {
           this.outputUtxos[utxo].appDataHash.toString() !== "0" &&
           this.outputUtxos[utxo].includeAppData
         )
-          throw new TransactionError(
-            TransactionErrorCode.UNIMPLEMENTED,
-            "encryptUtxos",
-            "Automatic encryption for utxos with application data is not implemented.",
+          console.log(
+            "Warning encrypting utxos with app data as normal utxo without app data. App data will not be encrypted.",
           );
+        // throw new TransactionError(
```

Comment:
Same question here - I guess we decided to warn instead of throwing an error?

Reply:
I want to implement this feature in the coming days that verifier 4in4out can store encrypted utxos then we will support it.
Right now I need this to not throw for the zk voting poc.

Reply:
I will leave the comment for now since we might want to reenable